testsuite/gateways/wasm/extension.py

```diff
@@ -106,14 +104,27 @@ def synchronise_mapping_rules(self):
         map_rules = config["content"]["proxy"]["proxy_rules"]
         self.add_mapping_rules(map_rules)
 
-    def add_credentials(self, credentials):
-        """Adds credential into the extension
-            credentials contains list of credentials with attributes:
+    def replace_credentials(self, credentials):
+        """Replaces previous credentials and deploys new into the extension.
+            `credentials` contains list of credentials with attributes:
             label: one of "user_key" or "app_id" or "app_key",
-            credential_location: one of "query_string", "header" or "authorization"
+            credential_location: one of "query_string", "header", "authorization", or "oidc"
             key: search key
         """
         ops = []
+        for label in ["user_key", "app_id", "app_key"]:
```

Comment:
[Minor] Use set

Reply:
Addressed

Reply:
It doesnt really make a difference and set usage generates lint warning 
` C0208: Use a sequence type when iterating over values (use-sequence-for-iteration)`
Reverting